notebooks/20220523_selecting_wt_ctrls.ipynb

```diff
@@ -0,0 +1,1526 @@
+       "22  NA                                         \n",
+       "23  NA                                         \n",
+       "24  NA                                         \n",
+       "25  NA                                         \n",
+       "26  NA                                         \n",
+       "27  NA                                         \n",
+       "28  NA                                         \n",
+       "29  NA                                         \n",
+       "30  NA                                         \n",
+       "⋮   ⋮                                          \n",
+       "72  NA                                         \n",
+       "73  NA                                         \n",
+       "74  NA                                         \n",
+       "75  NA                                         \n",
+       "76  NA                                         \n",
+       "77  NA                                         \n",
+       "78  NA                                         \n",
+       "79  NA                                         \n",
+       "80  NA                                         \n",
+       "81  NA                                         \n",
+       "82  NA                                         \n",
+       "83  NA                                         \n",
+       "84  NA                                         \n",
+       "85  NA                                         \n",
+       "86  NA                                         \n",
+       "87  NA                                         \n",
+       "88  NA                                         \n",
+       "89  NA                                         \n",
+       "90  NA                                         \n",
+       "91  NA                                         \n",
+       "92  NA                                         \n",
+       "93  NA                                         \n",
+       "94  NA                                         \n",
+       "95  NA                                         \n",
+       "96  NA                                         \n",
+       "97  NA                                         \n",
+       "98  NA                                         \n",
+       "99  NA                                         \n",
+       "100 NA                                         \n",
+       "101 NA                                         "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "runtable_pa14 %>%\n",
+    "  rowwise() %>%   # rowwise grouping\n",
+    "  filter(any(grepl(\"wt|wild type|wildtype|WT|ctrl|ontrol|LB|ysogeny\", c_across(starts_with(\"grepl\"))))) %>% # aiming for \"wild type\", but also matches \"growth\"\n",
+    "  select(experiment, run, bio_project, starts_with(\"grepl\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51eed6a3",
+   "metadata": {},
+   "source": [
+    "### PA14 notes\n",
```

Comment:
This huge chunk of text maybe feels like it belongs in a separate file? But if you have a good reason to keep it here, then that's fine.